### ISSUE TITLE
Cerebral subcotrical cyst

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10358]
**Preferred term label:**
Cerebral subcotrical cyst

**Synonyms**

**Definition (free text, please give PubMed ID)**
A fluid-filled sac (cyst) in the region below the cortex of the cerebrum. In magnetic resonance imaging, the fluid within the cyst has the same appearance as cerebrospinal fluid (CSF).

PMID:21977097

Comment: . Megalencephalic leukoencephalopathy with subcortical cysts is characterized by subcortical cysts that are present in the anterior temporal region most commonly and often in the frontoparietal region. Later, MRI may only reveal cerebral atrophy. The size and number of the cysts may increase over time. See Figure 1 of PMID:21977097

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 620447 
PMID: 37143309. 3/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
620448 
PMID: 37143309. 2/2

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000461
- label: Cerebral subcortical cyst
- parents: [Abnormal cerebral subcortex morphology (HP:0010993)](https://hpo.jax.org/app/browse/term/HP:0010993)
- pmid: PMID:21977097
- def: A fluid-filled sac (cyst) in the region below the cortex of the cerebrum. In magnetic resonance imaging, the fluid within the cyst has the same appearance as cerebrospinal fluid (CSF).
- comment: Megalencephalic leukoencephalopathy with subcortical cysts is characterized by subcortical cysts that are present in the anterior temporal region most commonly and often in the frontoparietal region. Later, MRI may only reveal cerebral atrophy. The size and number of the cysts may increase over time. See Figure 1 of PMID:21977097.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10358
- synonyms:

**Comment by kanems** [comment]
I think there is a typo in the preferred term label:
Subcortical (instead of subco_tri_cal)

**Comment by pnrobinson** [comment]
@kanems  -- thanks! I already caught this error! :-0
